grpc/server/src/backend.cpp

```diff
@@ -0,0 +1,56 @@
+#include "backend.h"
+
+#include <grpc/grpc.h>
+#include <grpc++/server.h>
+#include <grpc++/server_builder.h>
+#include <grpc++/server_context.h>
+#include <grpc++/security/server_credentials.h>
+
+#include "action/actionrpc_impl.h"
```

Comment:
So ideally, backend shall include all plugin header files as it doesn't know what a user wants. Is that right ?

Reply:
Ideally, plugins are somehow loaded dynamically. But the first version is hard-coded because it is faster to do xD.